cluster/vm-nocloud.yaml

```diff
@@ -21,7 +21,12 @@ spec:
   volumes:
     - name: registryvolume
       registryDisk:
-        image: kubevirt/cirros-registry-disk-demo:devel
+        image: kubevirt/fedora-cloud-registry-disk-demo:devel
```

Comment:
Once you add a test, make sure that you use cirros and bash scripts instead of fedora. It is unbearably slow.